implementations/rust/ockam/ockam_app/src/quit/menu_items.rs

```diff
@@ -10,7 +10,7 @@ pub struct QuitActions {
 
 impl QuitActions {
     pub fn new() -> QuitActions {
-        let quit = CustomMenuItem::new("quit".to_string(), "Quit").accelerator("cmd+q");
+        let quit = CustomMenuItem::new("quit".to_string(), "Quit Ockam").accelerator("cmd+q");//update label to "Quit Ockam"
```

Comment:
> //update label to "Quit Ockam"

Could you remove this comment please. The code doesn't need to talk about an old state.

Reply:
yeah, sure. @mrinalwadhwa